### ISSUE TITLE
Error running LanguageClassification

### ISSUE THREAD

**Issue by Samoed** [embeddings-benchmark/mteb#1108]
Hi! I just runned LanguageClassification and got error `KeyError: 'default'` in [this line](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/load_results/mteb_results.py#L179). It seems that this happen because of #1094. MtebResults is [expecting eval_langs to be language](https://github.com/embeddings-benchmark/mteb/blob/main/mteb/abstasks/TaskMetadata.py#L293) but getting `default`. I'm not shure why this happening

Code for this issue
```python
import mteb
from mteb.models.sentence_transformers_models import all_MiniLM_L6_v2

model = mteb.get_model(model_name=all_MiniLM_L6_v2.name, revision=all_MiniLM_L6_v2.revision)
tasks = mteb.get_tasks(tasks=["LanguageClassification"])
evaluation = mteb.MTEB(tasks)
evaluation.run(model, output_folder="results", verbosity=2)
```

**Comment by Samoed** [comment]
I rerun code in google collab and everything fine. Maybe I forgot to update my fork

**Comment by KennethEnevoldsen** [comment]
I actually believe this was the same issue as was fixed in #1094. Glad to hear that it works. Let us know if you experience any issues with it
